Makefile

```diff
@@ -13,7 +13,9 @@ lint:
 	poetry run ruff check allowed/allowed.py
 
 run_tests:
+	@echo "Tests in poetry environment, WITH ipython and ptype installed"
 	-@poetry run tests/tests.sh run
+	@echo "Tests in global environment, WITHOUT ipython or pytype installed"
```

Comment:
I know you have only altered the messaging here, and didn't add the code in this PR, but I have a couple comments:

Maybe we could mention in docs about the need to have "clean" global `python3.10` environment without `ipython` and `pytype` for tests to work as expected?

Is there a different way we can do this using `pyproject.toml`, dependency groups and the `Makefile` to ensure an environment is free from `ipython` and `pytype`?